src/screens/CurrentTrack/index.tsx

```diff
@@ -35,66 +38,52 @@ const parseAndPostWhoSampledData = async (artists: ArtistObjectSimplified[], nam
     return false;
   }
 };
-
-const useBedfellowService = () => {
-  const [isLoading, setIsLoading] = useState<boolean>(false);
-  const [trackSamples, setTrackSamples] = useState<BedfellowTrackSamples | null>(null);
-  const [refresh, setRefresh] = useState<boolean>(false);
-
-  const loadBedfellowData = async (artists: ArtistObjectSimplified[], track: string) => {
-    setIsLoading(true);
-
-    try {
-      if ((artists[0].name, track)) {
-        const trackSamplesResults = await Promise.all(
-          await artists.map(async (artist) => {
-            const result: BedfellowTrackSamples | null = await getBedfellowDBData(artist?.name, track);
-            return result;
-          })
-        );
-        // This could lead to a situation where we make a bunch of calls then only use the first result
-        // let's try to make this smarter later
-        const filteredTrackSampleResults = trackSamplesResults.filter((trackSample) => trackSample);
-        if (filteredTrackSampleResults.length) {
-          setTrackSamples(filteredTrackSampleResults[0]);
-          setIsLoading(true);
-          return;
-        }
-      }
-      const result = await parseAndPostWhoSampledData(artists, track);
-      if (result) {
-        const sampleData = await getBedfellowDBData(artists[0].name, track);
-        setTrackSamples(sampleData);
+const loadBedfellowData = async (artists: ArtistObjectSimplified[] = [], track: string = '') => {
+  try {
+    if ((artists[0].name, track)) {
```

Comment:
What is this syntax?

Reply:
fix, unsure how this got into there but I'm the sole author so better peep that mirror